crt_portal/cts_forms/templates/forms/widgets/crt_date_entry.html

```diff
@@ -0,0 +1,14 @@
+<div class="input-group date" id="{{ widget.datepicker_id }}" data-target-input="nearest">
+  {% include "django/forms/widgets/input.html" %}
+  <div class="input-group-append" data-target="#{{ widget.datepicker_id }}" data-toggle="datepicker">
+    <div class="input-group-text"><i class="fa fa-calendar"></i></div>
+  </div>
+</div>
+
+<script>
```

Comment:
We must have a `nonce="{{ request.csp_nonce }}` here for security reasons.

Reply:
done